inventory_management_system_api/routers/v1/unit.py

```diff
@@ -0,0 +1,25 @@
+"""
+Module for providing an API router which defines routes for managing Units using the `UnitService`
+service.
+"""
+
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends
+
+from inventory_management_system_api.schemas.unit import UnitSchema
+from inventory_management_system_api.services.unit import UnitService
+
+logger = logging.getLogger()
+
+router = APIRouter(prefix="/v1/units", tags=["units"])
+
+
+@router.get(path="/", summary="Get Units", response_description="List of Units")
```

Comment:
I take it that we don't need an endpoint for getting a unit by ID?

Reply:
Yes we don't. I will remove the  "/" here as well.